pkgs/by-name/sh/shopify-cli/package.nix

```diff
@@ -1,24 +1,33 @@
-{ buildNpmPackage, lib, testers, shopify-cli }:
-let
-  version = "3.69.3";
-in
-buildNpmPackage {
+{
+  lib,
+  testers,
+  shopify-cli,
+  fetchFromGitHub,
+  stdenv,
+  pnpm,
```

Comment:
Most likely you need to change `pnpm` to `pnpm_9`. Since #371832, it points to `pnpm_10` instead of `pnpm_9`.